src/e2e-test/java/io/cdap/plugin/gcp/tests/runner/TestRunner.java

```diff
@@ -24,7 +24,7 @@
  */
 @RunWith(Cucumber.class)
 @CucumberOptions(
-        features = {"src/e2e-test/features"},
+        features = {"src/e2e-test/features/DemoGcs.feature"},
```

Comment:
why do we have GCS test under plugin/gcp?

Reply:
This folder was created only for Demo.

Reply:
The demo is done now, so let's move code to the right place. 
Tests for a GCS should be under src/e2e-test/java/io/cdap/plugin/gcs

Reply:
Sure. We will update this in GCS testcases PR.